ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -214,26 +229,48 @@ TensorFilterLlamaCpp::invoke (const GstTensorMemory *input, GstTensorMemory *out
 }
 
 /**
- * @brief Invoke llamacpp using input tensors
+ * @brief Create output tensor
+ *
+ * The memory allocated for the output tensor data using strndup
+ * must be freed by the caller of this function to avoid memory leaks.
  */
 void
-TensorFilterLlamaCpp::invoke_dynamic (GstTensorFilterProperties *prop,
-    const GstTensorMemory *input, GstTensorMemory *output)
+TensorFilterLlamaCpp::createOutputTensor (const char *buf, int n,
+    GstTensorMemory *output, GstTensorFilterProperties *prop)
```

Comment:
take a look at this cpp-linter warning.

Reply:
Only changes to static functions are valid.